W-16224348-AltText-Mule47-dm (#2772)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mule-app-properties-system.adoc

. Pick the *Arguments* tab.
. Add your arguments to the *VM Arguments* field, prepending the property names with `-D`
+

> image::arguments-tab.png["Arguments tab in a run configuration showing settings for JVM arguments and environment variables"]

Author: ```suggestion
image::arguments-tab.png["Arguments tab showing settings for JVM arguments in Anypoint Studio"]
```
